tests/unittests/HyphenTest.cpp

```diff
@@ -323,8 +319,18 @@ struct HyphenNetwork {
 };
 } // namespace
 
+// Copy values from one PlaceholderBindings to another by name. This allows for
+// training a network on one EE then using the trained weights on another.
+void copyBetweenBindings(llvm::StringRef name, PlaceholderBindings &src,
```

Comment:
Isn't this exactly the same as `PlaceholderBindings::copyToTarget()`? Why is it necessary?